src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -43,6 +45,19 @@ public DiagnosticsTelemetryModule()
         }
 
         /// <summary>
+        /// Gets or sets the delay between heartbeats in milliseconds.
```

Comment:
Remove the "in milliseconds" part (unnecessary/misleading now that it's a `TimeSpan`)